CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,116 @@
+- CFPB events and meetings,
+- All of CFPB’s online forums and virtual collaboration tools, including code repositories
+
+
+## What we strive for
+
+At CFPB, we strive to create a welcoming and inclusive culture that empowers people to best protect 
+the financial interests of all consumers. That kind of atmosphere requires an open exchange of ideas 
+balanced by thoughtful guidelines. Examples of behavior that contributes to a positive environment 
+for our open source community include:
+
+- Demonstrating empathy and kindness toward other people
+- Being respectful of differing opinions, viewpoints, and experiences
+- Giving and gracefully accepting constructive feedback
+- Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+- Focusing on what is best not just for us as individuals, but for the overall community and public
+
+## Unacceptable behavior
+
+To help understand the kinds of behaviors that are illegal or run counter to the culture we seek to 
+foster, we've listed below actions that violate federal law and CFPB policy. We've also included 
+steps to take if you encounter behavior that runs contrary to this policy.
+
+The CFPB Policy Statement on Equal Employment Opportunity and Workplace Harassment forbids 
+discrimination or harassment based on:
+
+- Race
+- Color
+- Religion
+- Sex (including pregnancy, sexual orientation, transgender status, gender identity or expression, gender non-conformity, or sex stereotyping of any kind)
+- National origin
+- Disability
+- Age (40 years or older)
+- Genetic information
+- Parental status
+- Political affiliation
+- Marital status
+- Uniformed status
+- Membership in a labor organization or union activities
+- Prior equal employment opportunity (EEO) or whistleblower activity, or
+- Any other factor unrelated to your merit
+
+The policy also forbids harassing conduct, which includes unwelcome conduct based on any (or a combination of) protected traits or characteristics. Such conduct may take the form of any of the following: 
+
+- Offensive jokes, comments, objects, or pictures 
+- Questions about a person’s identity (e.g., disability status, gender identity, sexual orientation, national origin, etc.) 
+- Undue attention 
+- Ridicule or mockery 
+- Insults or put-downs 
+- Touching/physical contact 
+- Slurs or epithets 
+- Threats or other forms of intimidation 
+- Physical or sexual assault
+
+## Reporting violations
+
+If you are a CFPB employee, former CFPB employee, or job applicant to CFPB and believe you have been 
+discriminated against or harassed on the basis of race, color, religion, sex (including pregnancy, 
+sexual orientation, transgender status, gender identity or expression, gender non-conformity, or sex 
+stereotyping of any kind), national origin, disability, age (40 years or older), genetic information, 
+parental status or retaliated against for prior EEO activity, contact the CFPB’s EEO Office.
```

Comment:
```suggestion
or parental status, or retaliated against for prior EEO activity, contact the CFPB’s Office of Civil Rights.
```

Reply:
I think these other suggestions got lost when you made the change to spell out EEO.

Reply:
I'm going to add the comma, but keeping the "or" where it is since that is copied directly from EEO language used elsewhere